ember-phone-input/src/components/phone-input.ts

```diff
@@ -332,11 +376,11 @@ export default class PhoneInputComponent extends Component {
     };
   }
 
-  _onCountryChange() {
-    const selectedCountry = this._iti.getSelectedCountryData();
+  _onCountryChange(): void {
```

Comment:
Here applies a similar comment as the one I wrote here: https://github.com/qonto/ember-phone-input/pull/749/files#diff-34e5f38d343c0fe320d293f192edc4edb296ad67bbfce63e4fc87af453fed6eaR348

Reply:
Fixed